src/databricks/labs/ucx/hive_metastore/table_ownership.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
@nfx : The PR became big due to a required refactor to avoid circular import as explained in this docstring.

Reply:
that's fine